shell/edgex-publish-snap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Ensure we fail the job if any steps fail
+set -e -o pipefail
+snapcraft login --with $HOME/EdgeX
+
+# Build snap and grab the generated snap name
+SNAP_TO_PUSH=$(snapcraft | awk '/Snapped/ {print $2}')
+
+# Push the generated snap and grab the revision number
+REVISION=$(snapcraft push $SNAP_TO_PUSH | awk '/Revision/ {print $2}')
+
+# Release the snap to channel
+snapcraft release edgexfoundry $REVISION --channel=$SNAP_CHANNEL
```

Comment:
You don't need the `--channel=` here, you can just use
```
snapcraft release edgexfoundry $REVISION $SNAP_CHANNEL
```

(at least that's with the most recent version of snapcraft: `snapcraft, version 2.42.1+git25.25043ab`)

Reply:
Hmm I'm not sure which version we are on yet, will likely keep this unless it doesn't work.

Reply:
Sure, that's fine just at some future point this may not be supported as an option, not sure